### PR TITLE
Fix CVE-2023-34062

### PR DESCRIPTION
**Pull request by npepinpe**
## Description

This PR disables the default static resource mapping and bumps reactor-netty-http to 1.1.13 to fix CVE-2023-34062

## Related issues

closes SEC-768

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/camunda/zeebe/compare/stable/0.24...main?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/1.3`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark

Documentation:
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] If the PR changes how BPMN processes are validated (e.g. support new BPMN element) then the Camunda modeling team should be informed to adjust the BPMN linting.

Other teams:
If the change impacts another team an issue has been created for this team, explaining what they need to do to support this change.
- [ ] [Operate](https://github.com/camunda/operate/issues)
- [ ] [Tasklist](https://github.com/camunda/tasklist/issues)
- [ ] [Web Modeler](https://github.com/camunda/web-modeler/issues)
- [ ] [Desktop Modeler](https://github.com/camunda/camunda-modeler/issues)
- [ ] [Optimize](https://github.com/camunda/camunda-optimize/issues)

Please refer to our [review guidelines](https://github.com/camunda/zeebe/wiki/Pull-Requests-and-Code-Reviews#code-review-guidelines).

### CODE REVIEW

File: parent/pom.xml [hunk 1]
```diff
@@ -1094,6 +1102,18 @@
         <version>${version.identity}</version>
       </dependency>
 
+      <!--      JMH Tests   -->
```
Comment (megglos) [inline parent/pom.xml:1105]: ❓   how come this moved?

File: parent/pom.xml [hunk 2]
```diff
@@ -1073,7 +1081,7 @@
       <dependency>
         <groupId>io.github.acm19</groupId>
         <artifactId>aws-request-signing-apache-interceptor</artifactId>
-        <version>2.3.1</version>
+        <version>${version.aws-signing}</version>
```
Comment (megglos) [inline parent/pom.xml:1084]: ❓ that's just a tidying up a missed version property, but not related, right?

**Review by megglos — APPROVED** [review]
Thanks for acting on this right away! Just two questions to better understand some, I assume, unrelated changes. Pre-approved!

File: parent/pom.xml (hunk 1 above)
Comment (npepinpe) [reply parent/pom.xml:1105]: Same as below, just tidying things up and moving it above the dependency convergence section.

File: parent/pom.xml (hunk 2 above)
Comment (npepinpe) [reply parent/pom.xml:1084]: Exactly :+1:

**Comment by npepinpe** [comment]
bors merge

**Pull request merged**